core/src/main/java/bisq/core/api/CoreDisputeAgentsService.java

```diff
@@ -100,6 +100,10 @@ void registerDisputeAgent(String disputeAgentTypeString, String registrationKey)
                     signature = refundAgentManager.signStorageSignaturePubKey(Objects.requireNonNull(ecKey));
                     registerRefundAgent(nodeAddress, languageCodes, ecKey, signature);
                     return;
+                default:
+                    // Codacy requires default label in switch, Intellij does not.
+                    //noinspection UnnecessaryReturnStatement
+                    return;
```

Comment:
I think we can ignore codacy on this one going forward as it's possible to merge even when it's complaining. Perhaps we could even configure codacy to ignore it.

Reply:
Unused default switch label deleted in 0332711.